test/conf/services.json

```diff
@@ -159,7 +159,7 @@
              "destinationSubdir" : "basicplayapp",
              "cmd": [
                      "./basicplayapp/bin/basicplayapp",
-                     "-DProd.microservice.whitelist.useWhitelist=false",
+                     "-DProd.microservice.allowlist.useAllowlist=false",
```

Comment:
What does this actually do?
I had assumed that the basicplayapp would be using it somehow? This whole config section seems outdated, with -DProd too...
Can we just delete these lines?

Reply:
The corresponding change is at `test/unit/test_startcommands.py`.
It is simply testing that the configured start command, together with all of its arguments, is indeed used to start the service.

As there are other tests covering that the service starts, it doesn't matter what the application configuration parameter is, as long as it doesn't stop the application from running.

Reply:
The `basicplayapp` does not require any system parameters to start, so have removed these where unnecessary.
Where the test is explicitly testing that the configured command can contain system parameters, have used an "obviously dummy" system parameter name / value.

Note that `basicplayapp` is Play 2.2 :-)

Reply:
Play 2.2 😭